pkg/networkmanager/interfaces.js

```diff
@@ -2746,6 +2769,35 @@ PageNetworkInterface.prototype = {
                                 .syn_click(self.model, function () { configure() })));
             }
 
+            function render_general_settings_row() {
+                var parts = [];
+                var rows = [];
+                var options = settings.connection;
+
+                if (!options)
+                    return null;
+
+                parts.push("Auto-connection: " +
+                     (options.autoconnect ? "Enabled" : "Disabled"));
+
+                if (options.autoconnect)
+                    parts.push("(Priority: " + (options.autoconnect_priority >= 0
```

Comment:
We should generally only show "interesting" things in the summary. Thus "Priority: None" should be omitted entirely.